.github/workflows/publish_test_package.yml

```diff
@@ -0,0 +1,71 @@
+name: Publish test package
+on: 
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+jobs:
+  publish-test-package:
+    # This action only runs on forks because it pushes a new release.
+    # It is not wanted on the main repository as it would spam the releases.
+    if: ${{ github.repository != 'canalplus/rx-player' }}
+
+    runs-on: [ubuntu-latest]
+    permissions:
+      packages: write
+      contents: write
+      pull-requests: write
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 20
+          scope: "@${{ github.repository_owner }}"
+          registry-url: "https://npm.pkg.github.com"
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+ 
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Build Library
+        run: npm run build
+
+      - name: Create Zip file
+        run: echo "npm-pack=$(npm pack)" >> $GITHUB_OUTPUT
+        id: npm-pack
+
+      - run : "echo ${{ github.sha }}"
+      
+      - name: Upload Zip to release
+        uses: svenstaro/upload-release-action@v2
+        id: upload-action
+        with:
+          repo_token: ${{ secrets.GITHUB_TOKEN }}
+          file: ${{ steps.npm-pack.outputs.npm-pack }}
+          asset_name: rx-player.tgz
+          tag: "preview.${{ github.sha }}"
+          release_name: "preview.${{ github.sha }}"
+          overwrite: true
+          body: "This is my release text"
```

Comment:
To update. Would be nice to put the commit HASH + branch name + date here somewhere, in a text purposed for humans (i.e. makes sense when read)